src/uct/ib/configure.m4

```diff
@@ -218,6 +222,10 @@ AS_IF([test "x$with_ib" = "xyes"],
               AC_CHECK_DECLS([ibv_alloc_td],
                       [has_res_domain=yes], [], [[#include <infiniband/verbs.h>]])])
 
+       AS_IF([test "x$with_devx" != xno], [
+            AS_IF([test "x$ac_cv_have_decl_MLX5DV_CONTEXT_FLAGS_DEVX" = xyes], [
```

Comment:
would be better to check for decl MLX5DV_CONTEXT_FLAGS_DEVX only if $with_devx!=no, or MLX5DV_CONTEXT_FLAGS_DEVX could be defined when devx is disabled, which is not very consistent